R/NimbleAppend.R

```diff
@@ -0,0 +1,55 @@
+#' @import Seurat tidyr Matrix
+
+AppendNimbleCounts <- function(seuratObject, file, appendToCounts=FALSE) {
+  if (!file.exists(file)) {
+    stop("Nimble file not found.")
+  }
+  
+  # Read file and construct df
+  df <- read.table(file, sep="\t", header=FALSE)
+  
+  # Remove blank feature names, just in case
+  df <- df[!(df$V1==""), ]
+  
+  #Remove ambiguous features
+  df <- df[!grepl(",", df$V1), ]
```

Comment:
This is a key point where I would do extra logging. Such as:

```
toDrop <- df[grepl(",", df$V1), ]
if (nrow(toDrop) > ) {
 print(paste0('A total of ', length(unique(toDrop$V1)), ' ambiguous features will be dropped'))
  print(sort(table(toDrop$V1)))
}
```

Reply:
Also, I would add an argument for 'dropAmbiguousFeatures = TRUE', to let the user toggle this. For now, let's just always drop, but good to have options.